src/map/script_constants.hpp

```diff
@@ -9488,6 +9488,10 @@
 	export_constant(CPC_CHAR);
 	export_constant(CPC_ACCOUNT);
 
+	/* instance_warpall flags */
+	export_constant(IWA_NOTDEAD);
```

Comment:
```suggestion
	export_constant(IWA_DEAD);
```
Sorry I forgot that one!